docs/src/wallet/app-wallets.md

```diff
@@ -1 +1,26 @@
-Documentation of supported for App Wallets coming soon!
\ No newline at end of file
+# App Wallets
+Solana is supported by multiple third-party apps which should provide a familiar
+experience for most people who are new or experienced with using crypto wallets.
+
+## Trust Wallet
+[Trust Wallet](https://trustwallet.com/) is an app for iOS and Android.  This is
+currently the easiest and fastest way to get set up with a new wallet on Solana.
+The app is free and getting your wallet set up only takes a few minutes.
+
+{% page-ref page="trust-wallet.md" %}
+
+## Ledger Live with Ledger Nano S
+[Ledger Live](https://www.ledger.com/ledger-live) is available as free desktop
+software and as a free app for iOS and Android.  It is used to manage apps and
+crypto accounts on a Ledger *hardware wallet*, which must be purchased
```

Comment:
Only use italics when introducing a word.  And when you're introducing word, explain what it is.  In this case, you already explained what this word is in the README.

Reply:
fixed